mne/preprocessing/ica.py

```diff
@@ -658,6 +658,27 @@ def _transform_evoked(self, evoked):
 
         return sources
 
+    def get_components(self, components=None):
```

Comment:
You mean because `get_components(components=x)` always equals `get_components()[x]`?..

Yeah I guess the `components` arg is 100% useless :D